src/main/java/uk/gov/hmcts/juror/api/moj/repository/JurorPoolRepository.java

```diff
@@ -58,4 +58,9 @@ JurorPool findByOwnerAndJurorJurorNumberAndPoolPoolNumberAndIsActive(String owne
 
     @Query(value = "SELECT * from juror_mod.jsm_report_by_pool(:poolNumber)", nativeQuery = true)
     String getJsmReportByPool(@Param("poolNumber") String poolNumber) throws SQLException;
+
+    @Query(value = "SELECT * from juror_mod.jsm_report_by_court(:locCodes, :fromDate, :toDate)", nativeQuery = true)
+    String getJsmReportByCourt(@Param("locCodes") String poolNumber,
```

Comment:
can jsm be fully quantified? (makes reading the code later easier as JSM wont mean anything but jurySummoningReport will :))

Reply:
have updated as suggested